doc-gen/src/logger.ts

```diff
@@ -0,0 +1,36 @@
+import ora from "ora";
+import pc from "picocolors";
+import { format as utilFormat } from "util";
+
+function format(args: any[]) {
+  return utilFormat(...args)
+    .split("\n")
+    .join("\n");
```

Comment:
What does this split/join do?

Reply:
huh... good question... I think I ripped this from a wagmi cli file at somepoint. I have 0 clue why this is here. I'll remove it cuz it doesn't really make sense